.buildkite/full/react-native-android-pipeline.full.yml

```diff
@@ -149,6 +151,8 @@ steps:
           - "0.72"
           - "0.73"
           - "0.74"
```

Comment:
according to the 0.77 [release notes](https://reactnative.dev/blog/2025/01/21/version-0.77) (at the very bottom) v0.74 is now unsupported. should we drop these versions from our test matrix?

Reply:
Discussed in Slack - I've now created a separate task to update the test matrix 👍🏾